src/RoveComm/RoveCommTCP.h

```diff
@@ -0,0 +1,92 @@
+/******************************************************************************
+ * @brief The RoveCommTCP class is used to send and receive data over a TCP
+ *        connection. This class is a subclass of AutonomyThread, so it can be
+ *        run in its own thread.
+ *
+ * @file RoveCommTCP.h
+ * @author Eli Byrd (edbgkk@mst.edu)
+ * @date 2024-02-06
+ *
+ * @copyright Copyright Mars Rover Design Team 2024 - All Rights Reserved
+ ******************************************************************************/
+
+#ifndef ROVECOMM_TCP_H
+#define ROVECOMM_TCP_H
+
+#include <arpa/inet.h>
```

Comment:
Put implicit include in 
/// \cond
#include <array>

/// \endcond
to avoid them being included in doxygen graph.